account_invoice_check_total/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Invoice Check Total',
+    'summary': """
+        Check if the verification total is equal to the bill's total""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Acsone SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
+    'depends': [
+        'account',
+    ],
+    'data': [
```

Comment:
Sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L27)

Reply:
@nikul-serpentcs Done